.github/workflows/summary-labeler.yml

```diff
@@ -0,0 +1,16 @@
+name: Summary labeler
+
+on:
+  pull_request:
```

Comment:
`pull_request` does not have permission to edit labels. Maybe change to `pull_request_target` instead?

Reply:
Give me a sec I was just googling the error messages

Reply:
Well this sucks, having to merge this to test it isn't ideal, I'll probably be at work when or if ever this gets merged, it would be quite frustrating if it breaks and someone has to step up and fix my stuff